media-libs/openjpeg/openjpeg-2.1.2.recipe

```diff
@@ -85,5 +86,7 @@ INSTALL()
 
 TEST()
 {
-	make check
+	cd tests
+	cmake ..
+	make
```

Comment:
The tests require an extra file that is not in the source tarball, so I've removed the **`TEST()`** function.

BTW, I think that if we ever chose to add the data set used for the tests then we should also pass «**`-DBUILD_TESTING=ON`**» to the call to **`cmake`** in **`BUILD()`** and define **`TEST()`** with something like:

```
TEST()
{
	cd tests
	cmake \
		-DOPENJPEG_SOURCE_DIR=$sourceDir \
		-DOPENJPEG_BINARY_DIR=$sourceDir/build \
		-DZ_INCLUDE_DIRNAME=$portPackageLinksDir/devel~libz$secondaryArchSuffix/$relativeIncludeDir \
		-DPNG_INCLUDE_DIRNAME=$portPackageLinksDir/devel~libpng$secondaryArchSuffix/$relativeIncludeDir \
		-DTIFF_INCLUDE_DIRNAME=$portPackageLinksDir/devel~libtiff$secondaryArchSuffix/$relativeIncludeDir \
		.
	make
}
```